datafusion/core/src/physical_plan/planner.rs

```diff
@@ -462,13 +462,13 @@ impl DefaultPhysicalPlanner {
     ) -> BoxFuture<'a, Result<Arc<dyn ExecutionPlan>>> {
         async move {
             let exec_plan: Result<Arc<dyn ExecutionPlan>> = match logical_plan {
-                LogicalPlan::TableScan (TableScan {
-                    source,
-                    projection,
-                    filters,
-                    fetch,
-                    ..
-                }) => {
+                LogicalPlan::TableScan(TableScan {
+                                           source,
+                                           projection,
+                                           filters,
+                                           fetch,
+                                           ..
+                                       }) => {
```

Comment:
I'm curious why there are formatting changes like this that are unrelated to making BinaryExpr a struct. Were these changes made intentionally?

Reply:
They are auto-formatted by CLion IDE, let me try to to revert these unrelated changes. CC @jackwener